.github/workflows/tox.yml

```diff
@@ -0,0 +1,74 @@
+# yamllint disable rule:line-length
+name: tox
+on:  # yamllint disable-line rule:truthy
+  - pull_request
```

Comment:
This way, it will run only on PRs, but you added an icon in the readme, which shows CI Testing on master?

Reply:
I have no idea what the badge should look like.  If you have a working example I would be grateful.  I just copied and pasted what the Action link generated.

Reply:
I have an example from ansible-sshd:
![Run tests on CentOS 8](https://github.com/willshersystems/ansible-sshd/workflows/Run%20tests%20on%20CentOS%208/badge.svg)